src/main/java/nextstep/ladder/domain/entity/Line.java

```diff
@@ -24,15 +20,10 @@ public int size() {
   }
 
   public int move(int startingPoint) {
-    return IntStream.range(INIT, points.size() + INIT)
-                    .filter(index -> index  == startingPoint)
-                    .map(now -> {
-                      Point nowPoint = points.get(now);
-                      int direction = nowPoint.findDirection(points.get(now + LEFT));
-                      return now + direction;
-                    })
-                    .findFirst()
-                    .orElseThrow(IllegalArgumentException::new);
+    return points.stream()
+                 .filter(point -> point.isSame(startingPoint))
+                 .map(Point::move)
+                 .findFirst()
+                 .orElseThrow(IllegalArgumentException::new);
```

Comment:
```suggestion
    return points.get(startingPoint).move();
```
이렇게 변경이 되었다면 더 좋았을 것 같아요. :)